test/data/ansible/check_ostree.yaml

```diff
@@ -63,7 +63,8 @@
         - set_fact:
             total_counter: "{{ total_counter | int + 1 }}"
       rescue:
-        # TODO: gather Ignition logs
+        - name: Gather Ignition logs
+          shell: journalctl --identifier=ignition --all
```

Comment:
nitpick: would it be possible to check logs for a specific unit file using the `-u <unit>` option, instead of relying on syslog identifier? Not a big deal though.

Reply:
I agree with this but as Tomas said, not a big issue.  Should we dismiss this?